example/example-integration.js

```diff
@@ -14,22 +14,28 @@
    * @param {mixed} options Hash of obtions for the exampleAds plugin.
    */
   vjs.plugin('exampleAds', function(options){
-    
+
     var
-      
+
       player = this,
-      
+
       // example plugin state, may have any of these properties:
       //  - inventory - hypothetical ad inventory, list of URLs to ads
       //  - lastTime - the last time observed during content playback
       //  - adPlaying - whether a linear ad is currently playing
       //  - prerollPlayed - whether we've played a preroll
       //  - midrollPlayed - whether we've played a midroll
+      //  - postrollPlayed - whether we've played a postroll
       state = {},
-      
+
+      // just like any other video.js plugin, ad integrations can
+      // accept initialization options
+      adServerUrl = (options && options.adServerUrl) || "inventory.json",
```

Comment:
`&&` returns the second value when it succeeds, which is `options.adServerUrl` in this case. This line is being overly clever.